tools/flaky-test-reporter/main.go

```diff
@@ -26,15 +26,22 @@ import (
 	"os"
 
 	"github.com/knative/test-infra/shared/prow"
+	ftrConfig "github.com/knative/test-infra/tools/flaky-test-reporter/config"
 )
 
 func main() {
 	serviceAccount := flag.String("service-account", os.Getenv("GOOGLE_APPLICATION_CREDENTIALS"), "JSON key file for GCS service account")
 	githubAccount := flag.String("github-account", "", "Token file for Github authentication")
 	slackAccount := flag.String("slack-account", "", "slack secret file for authenticating with Slack")
+	configPath := flag.String("configfile", "", "Config file for overriding default config file")
```

Comment:
can we add the path to the default config here instead of empty?

Reply:
this is done purposely, so that when another tool wants to use the `config` package it doesn't need to understand where default config is

Reply:
This default is for the ftr tool since this is in main.go and not the config pkg? I agree that the config pkg should be generic, but why cant we add default here when defineing the flag?

Reply:
agreed and done